queries.sql

```diff
@@ -106,4 +106,71 @@ AND animals.escape_attempts=0;
 /* Who owns the most animals? */
 SELECT owners.full_name, COUNT(*) as count FROM owners join animals ON animals.owner_id=owners.id
 GROUP BY full_name
-ORDER BY count DESC
\ No newline at end of file
+ORDER BY count DESC
+
+
+/* Who was the last animal seen by William Tatcher? */
+SELECT visit.animalName, visit.date_of_visit FROM (SELECT vets.name as vetName, animals.name as animalName, visits.date_of_visit
+FROM animals 
+JOIN visits on (animals.id=visits.animals_id)
+JOIN vets on (vets.id=visits.vets_id)) as visit
+WHERE vetName='William Tatcher'
+ORDER BY date_of_visit DESC
+LIMIT 1;
```

Comment:
Optional

- [ ] Use select `animals.name` from visits table not animals even WHERE should be vets.name = 'William Tatcher'